server/notification-providers/sevenio.js

```diff
@@ -0,0 +1,79 @@
+const NotificationProvider = require("./notification-provider");
+const Axios = require("axios");
+const { DOWN, UP } = require("../../src/util");
+
+class SevenIO extends NotificationProvider {
+
+    name = "SevenIO";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+
+        const data = {
+            to: notification.sevenioTo,
+            from: notification.sevenioSender || "Uptime Kuma",
+            text: msg,
+        };
+
+        const axios = Axios.create({
+            baseURL: "https://gateway.seven.io/api/",
+            headers: {
+                "Content-Type": "application/json",
+                "X-API-Key": notification.sevenioApiKey,
+            },
+        });
```

Comment:
Lets not make introduce a different way of using axios

```suggestion
        const config = {
            baseURL: "https://gateway.seven.io/api/",
            headers: {
                "Content-Type": "application/json",
                "X-API-Key": notification.sevenioApiKey,
            },
        };
```